module/VuFind/src/VuFind/Auth/OpenIDConnect.php

```diff
@@ -360,6 +364,50 @@ public function getSessionInitiator(string $target): ?string
         return $this->getProvider()->authorization_endpoint . '?' . http_build_query($params);
     }
 
+    /**
+     * Perform cleanup at logout time.
+     *
+     * @param string $url URL to redirect user to after logging out.
+     *
+     * @return string Redirect URL (modified for OpenIDConnect logout).
+     */
+    public function logout($url)
+    {
+        $redirectUrl = $url;
+        $end_session_endpoint = false;
```

Comment:
Minor detail, but `$end_session_endpoint` should be `$endSessionEndpoint`

Reply:
See discussion above.

Reply:
Fixed now.